v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-a-wallet.md

---
index: 17
title: What is a wallet?

> description: Crypto wallets provide people with access to their tokens and the ability to make transactions with them.

Reviewer: ```suggestion
description: Crypto wallets provide access to your tokens and the ability to make transactions with them.
```

  - blockchain
  - atom
---

> A crypto wallet is like a debit card. Debit cards do not store funds but instead, they give people access to their funds wherever they choose to spend them. Wallets in the blockchain universe similarly do not store funds but instead provide people with access to their tokens and the ability to make transactions with them.

Reviewer: ```suggestion
A crypto wallet is like a debit card. Debit cards do not store funds but instead, they give you access to your funds wherever you choose to spend them. Wallets in the blockchain universe similarly do not store funds but instead provide you with access to their tokens and the ability to make transactions with them.
```
